### ISSUE TITLE
Binstar client does not have LDAP support

### ISSUE THREAD

**Issue by JeremyEnns** [Anaconda-Server/anaconda-client#161]
We added LDAP support to Binstar Server so Enterprise customers can use their existing authentication system to access the Binstar Server but we do not seem to have the same functionality in the binstar client, it still requires users to login with a local account not their LDAP account.

This seems like a missed requirement / feature request. 

Notes from Customer:

After changing the anaconda server configuration to use the LDAP based auth, I'm experiencing an unexpected behavior when trying to login using the command line binstar client.

bash-4.1$ binstar login 
Using binstar api site http://hostname:8080/api 
Username: vianeri1 
users's Password: <my ldap password> 
[error] Invalid Username password combination, please try again 
users's Password: <my old local password> 
login successful

Basically, it looks like that the current LDAP auth settings are ignored when processing the requests from the binstar client, and login is still only allowed with the older password, that I presume is still hashed in the users database.

At the same time I can regularly login into the browser gui using my ldap credentials, is this a known problem?

**Comment by tacaswell** [comment]
We have also seen this issue at BNL, but have not had the bandwidth to raise an issue about it.

**Comment by stephenakearns** [comment]
@JeremyEnns - just double-checking - is this still an open issue? (We are assuming it is still an issue and if yes, is a big issue).

**Comment by JeremyEnns** [comment]
@stephenakearns I was told that the anaconda-client (binstar-client) did not have any code to use LDAP authentication, so this is likely still an issue.  The client was originally created to interface with anaconda.org but since we are using it for Anaconda Server as well it needs to share the same feature set as Anaconda Server does.  If we add LDAP authentication features to the web interface then we also need to add them to the anaconda-client as well.

If you have an Anaconda Server setup already using LDAP, I can test the most recent client against it but I suspect this has not changed.

**Comment by srossross** [comment]
>  I was told that the anaconda-client (binstar-client) did not have any code to use LDAP authentication, so this is likely still an issue.

@JeremyEnns the anaconda-client does not use LDAP authentication because it authenticates with anaconda-server directly, if the server is using LDAP then the client is indirectly as well.

I think that the there was [a fix](https://github.com/Anaconda-Server/anaconda-server/commit/c051c878e390c3cddfe41c9dabf3e56b49891046) to the server that addressed this issue.

**Comment by srossross** [comment]
Closing issue, please re-open if required
